docs/docsite/rst/shared_snippets/basic_concepts.txt

```diff
@@ -47,23 +60,36 @@ A special form of a Task, that only executes when notified by a previous task wh
 Modules
 =======
 The code or binaries that Ansible copies to and executes on each managed node (when needed) to accomplish the action defined in each Task.
+
 Each module has a particular use, from administering users on a specific type of database to managing VLAN interfaces on a specific type of network device.
+
 You can invoke a single module with a task, or invoke several different modules in a playbook.
 Ansible modules are grouped in collections. For an idea of how many collections Ansible includes, see the :ref:`list_of_collections`.
 
 
 Plugins
 =======
-Pieces of code that expand Ansible's core capabilities, they can control how you connect to a managed node (connection plugins),
+Pieces of code that expand Ansible's core capabilities.
+
+Plugins can control how you connect to a managed node (connection plugins),
 manipulate data (filter plugins) and even control what is displayed in the console (callback plugins).
+
 See :ref:`working_with_plugins` for details.
 
 
 Collections
 ===========
-A format in which Ansible content is distributed that can contain playbooks, roles, modules, and plugins. You can install and use collections through `Ansible Galaxy <https://galaxy.ansible.com>`_. To learn more about collections, see :ref:`collections`. Collection resources can be used independently and discretely from each other.
+A format in which Ansible content is distributed that can contain playbooks, roles, modules, and plugins.
+
+You can install and use collections through `Ansible Galaxy <https://galaxy.ansible.com>`_.
+
+To learn more about collections, see :ref:`collections`.
+
+Collection resources can be used independently and discretely from each other.
 
 
 AAP
```

Comment:
How about if we remove mentions of AAP here while we're at it? I'm not sure how well this qualifies as an "Ansible concept".

Reply:
Done, i think the ref on the docs.ansible.com (already there) is enough.
PTAL